apps/docs/src/app/platform/component-docs/platform-forms/step-input/platform-step-input-examples/platform-number-step-input-template-example.component.html

```diff
@@ -0,0 +1,27 @@
+<main class="fd-page">
+    <div class="fd-page__content fd-has-background-color-neutral-2">
+        <fdp-form-group #formGroup [multiLayout]="true">
+            <fdp-form-field
+                #formField
```

Comment:
you dont probably need this handle #formField as you are not accessing anything in there.

Reply:
removed it